src/js/build.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/bash
 
 root=$(git rev-parse --show-toplevel)
-rm -rf "$root"/build/node/output/*
```

Comment:
does this affect the open-source build?

Reply:
Oh interesting, I didn't know that we make a build command for coprocessor, but I think it should be another command that saves the build result in the same folder. like `npm run build` and change the current command to `npm run build:v`, what do you think?

Reply:
example: 
`npm run build` for open source, and it compiles the ts and saves the result in `src/js/dist`
`npm run build:v` that compiles everything and save it on `vbuild/node/output`

Reply:
Since this is not currently part of the `build.sh` script, I'd say that if we make it we should do it the right way.
Depending on directories used by vtools outside of vtools commands is a sure way for errors. I'd propose that the build script default to an output directory and have a way to override it ( which is what the vtools command would do ?).

How does that sound?

Reply:
I was thinking that it might not work for users that are building redpanda with the `./build.sh` script in the root directory since that uses `build` and not `vbuild` as the output build directory.

Reply:
checking the redpanda readme and `build.sh`, this PR doesn't impact the open-source community, this change is focusing on `vtools`, when we add officially coprocessor to redpanda, we would modify the `build.sh` script in order to build coprocessor on `build/` folder.

Reply:
I change the code in order to support the open source community and vtools